app/controllers/provider/admin/user/personal_details_controller.rb

```diff
@@ -34,6 +35,7 @@ def current_password_verification
 
     unless current_user.authenticated?(user_params[:current_password])
       flash.now[:error] = 'Current password is incorrect.'
+      AuditLogService.call("User tried to change password: #{current_user.id}/#{current_user.username}") if user_params[:password].present?
```

Comment:
In this case, I'd add "tried, but failed due to incorrect password". Because it's clear that they tried, but not clear if they succeeded.

Reply:
Done here: https://github.com/3scale/porta/pull/3885/commits/3279c2334d276b700340b4c528937d41f70e4c4b